ang/civicase/Utils.js

```diff
@@ -236,39 +236,59 @@
   });
 
   module.factory('getActivityFeedUrl', function ($route, $location, $sce) {
-    return function (caseId, category, statusType, status, id) {
-      caseId = parseInt(caseId, 10);
-      var af = {};
+    return function (urlParams) {
```

Comment:
As this service is heavily changed, lets also add unit tests.

Reply:
I added the tests for the service. It's in the last commit.